### DISCUSSION TITLE
Sub-Dashboards

### DISCUSSION THREAD

**Discussion by quartzeye** [linuxserver/Heimdall#672] [Ideas]
There needs to be a way of having nested or sub-dashboards.  The ability to drop an icon/app on the dashboard that opens up a sub or nested iteration so that apps can be organized hierarchically.

**Comment by Node815** [comment]
I've created sub dashboards by using tags.  You can then link a generic link to the tab name it as you see fit.   For example, I have tabs for System administration which holds my Webmin  Links, Media tools which holds my media server stuff, Docker management, which contains my portainer instances and so forth.  This allows me to keep things a bit more organized.

**Reply by ka-mp-6765** [reply]
I'd like to revive this. I still think nesting is a good idea. With the current tagging it's still a lot on each page.
better overviews can be created by nesting. t.ex.
